envs/r.yaml

```diff
@@ -9,7 +9,7 @@ dependencies:
   - r-stringdist
   - r-dplyr=0.7.6
   - r-mvtnorm
-  - r-seurat=2
+  - r-seurat
```

Comment:
this should best be `r-seurat=3` to make it more explicit

Reply:
done #616c160